pkg/virt-controller/network/ipamclaims/OWNERS

```diff
@@ -0,0 +1,3 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+approvers:
+  - maiqueb
```

Comment:
IMHO we should also add a reviewers section. 

Let's make all three of us reviewers here. I'd also have us 3 - or at least 2 of us - as approvers.